### PR TITLE
Remove dependency on internal pip APIs

### LINKED ISSUE #111: Build is broken with pip 20.1
```
Collecting django-mock-queries
  Downloading django_mock_queries-2.1.4.tar.gz (19 kB)
    ERROR: Command errored out with exit status 1:
     command: ~/.pyenv/versions/3.5.9/envs/venv/bin/python3.5 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/zx/2wg3gb553mzf9qmv349dbczr0000gp/T/pip-install-a_7srd3z/django-mock-queries/setup.py'"'"'; __file__='"'"'/private/var/folders/zx/2wg3gb553mzf9qmv349dbczr0000gp/T/pip-install-a_7srd3z/django-mock-queries/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/zx/2wg3gb553mzf9qmv349dbczr0000gp/T/pip-pip-egg-info-wafekgr9
         cwd: /private/var/folders/zx/2wg3gb553mzf9qmv349dbczr0000gp/T/pip-install-a_7srd3z/django-mock-queries/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/zx/2wg3gb553mzf9qmv349dbczr0000gp/T/pip-install-a_7srd3z/django-mock-queries/setup.py", line 9, in <module>
        req = [str(ir.req) for ir in install_req]
      File "/private/var/folders/zx/2wg3gb553mzf9qmv349dbczr0000gp/T/pip-install-a_7srd3z/django-mock-queries/setup.py", line 9, in <listcomp>
        req = [str(ir.req) for ir in install_req]
    AttributeError: 'ParsedRequirement' object has no attribute 'req'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

from https://github.com/pypa/pip/issues/2286#issuecomment-68285791

> @dstufft: A setup.py which is importing pip is a broken setup.py already. We do not offer a programmatic API for people to use nor do we offer any backwards compatibility guarantees for any of the functions inside of pip.

### PR DESCRIPTION
**Pull request by rbusquet**
Fixes #111

### CODE REVIEW

**Comment by rbusquet** [comment]
@stphivos I don't understand why it's failing for python 2.

File: setup.py [hunk 1]
```diff
@@ -44,5 +36,10 @@ def read_md(filename):
     ],
     keywords='django orm mocking unit-testing tdd',
     packages=['django_mock_queries'],
-    install_requires=req,
+    install_requires=[
+        "mock",
+        "Django",
+        "djangorestframework",
+        "model-bakery==1.0.2",
+    ],
```
[the remark this answers is not in the document]
Comment (rbusquet) [reply setup.py:44]: Nice catch. I thought that the list of requirements is so small that it wouldn’t warrant a need for reading the requirements file.

I was even thinking this module shouldn’t really need to enforce the installation of any of these libraries, but then that’s out of the scope here.

**Comment by stphivos** [comment]
Thanks @rbusquet . It seems the build was stuck so I did a rebuild and everything passed.

**Comment by rbusquet** [comment]
Thanks @stphivos!!

**Pull request merged**